### PR TITLE
Create VitalyForm.vue

### PR DESCRIPTION
**Pull request by vitaly-savin**

### CODE REVIEW

File: components/ToDoList.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<template>
+    <v-container>
+        <v-card>
+            <h1 
+                :style="{
+                    color: inputValue.length < 5 ?
```
Comment (leresolu) [inline components/ToDoList.vue:6]: please move logic to computed value

File: components/ToDoList.vue [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        <v-card
+            v-if="notes.length !== 0"
+            class="py-5">
+            <ul class="list">
+                <li 
+                    v-for="(item, idx) in notes" 
+                    :key="item"
+                    class="list-item my-2">
+                    ({{ idx + 1 }})
+                     {{toUpperCase(item)}}
```
Comment (leresolu) [inline components/ToDoList.vue:41]: Please move entire content to computed value

File: components/ToDoList.vue [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                elevation="2"
+                rounded
+                class="my-3"
+                @click="addNewToDo">
+                Ajouter d'affaire
+            </v-btn>
+            <hr class="my-3" />
+        </v-card>
+        <v-card
+            v-if="notes.length !== 0"
```
Comment (leresolu) [inline components/ToDoList.vue:33]: please create a computed value "hasNotes"

File: components/ToDoList.vue [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                     <v-btn
+                        color="red accent-4"
+                        elevation="2"
+                        rounded
+                        class="my-3"
+                        @click.stop="removeToDo(idx)"
+                    >Delete</v-btn>
+                </li>
+                <hr />
+                <li>Nombre total de tâches : {{ notes.length }}</li>
```
Comment (leresolu) [inline components/ToDoList.vue:51]: please make a computed value "qtyNotes"

**Review by PanteaMartia — APPROVED** [review]
Good job Vitaly

**Review by leresolu — CHANGES_REQUESTED** [review]
Good job just a minor few things

File: components/ToDoList.vue (hunk 1 above)
Comment (gffilho) [reply components/ToDoList.vue:6]: Please, could you show me how to dot it? Thank you.

Comment (leresolu) [reply components/ToDoList.vue:6]: yes here is an example: 
```
computed: {
    h1Color () {
        return this.inputValue.length < 5 ? 'darkred' : 'green';
    }
}
```

and then in your template `style="{ color: h1Color }"`

File: components/ToDoList.vue [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        toUpperCase(item) {
+            return item.toUpperCase();
+        }
+    }
+}
+</script>
+
+<style scoped>
+.form-control input {
+    background-color: aqua;
```
Comment (leresolu) [inline components/ToDoList.vue:92]: hard to see with that color! use v-text-field instead maybe?

**Pull request merged**